<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleBasedTopologyDockWidgetBase</class>
 <widget class="QDockWidget" name="RuleBasedTopologyDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rule Based Topology</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QComboBox" name="langComboBox"/>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="rulePlayground">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	padding-top:2px;
	padding-bottom:2px;
}

QComboBox::drop-down {
    border: 0px; 
}

QComboBox::down-arrow {
    width: 10px;
    height: 10px;
}

QComboBox{
	border:0px;
	padding-bottom:1px;
	border-bottom: 0.5px solid rgb(234, 234, 234);
}


QGroupBox{
	background-color:white;
	border:0px;
	border-left:1px solid rgb(234, 234, 234);
	padding-left:2px;
	padding-top:0px;
	margin-top:0px;
}

QListWidget:item{
 	border-bottom: 0.5px solid rgb(234, 234, 234);
 }

QLineEdit{
	border: none;
	border-bottom: 0.5px solid rgb(234, 234, 234);
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="rulesTab">
       <attribute name="title">
        <string>Rules</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="rulesList">
          <property name="styleSheet">
           <string notr="true">
             QListWidget:item{border-bottom: 0.5px solid rgb(234, 234, 234)}
             QListWidget::item:hover{background-color: rgb(234, 234, 234)}
           </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="checkButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border: none;
	background-color: rgb(200, 200, 200);
}

QPushButton:hover{
	border: none;
	background-color:rgb(186, 189, 182);
}</string>
          </property>
          <property name="text">
           <string>Check</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="addRuleTab">
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
       <attribute name="title">
        <string>+</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="addRuleLayout">
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="titleLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="titleLabel">
            <property name="styleSheet">
             <string notr="true">Qlabel {
	color: rgb(186, 189, 182)
}</string>
            </property>
            <property name="text">
             <string>Rule title</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="layerStatus1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: white;
	background-color:white;
	border: 0px;
}</string>
            </property>
            <property name="text">
             <string>!</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QFormLayout" name="layerLayout1">
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="layerLabel1">
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
                <underline>true</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="text">
               <string>Layer</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="layerComboBox1">
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="currentText">
               <string/>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="fieldFilterStatus1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: white;
	background-color:white;
	border: 0px;
}</string>
            </property>
            <property name="text">
             <string>!</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QFormLayout" name="fieldFilterLayout1">
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="fieldFilterLabel1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">color:white</string>
              </property>
              <property name="text">
               <string>Layer:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGroupBox" name="fieldFilterGroupBox1">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="title">
               <string/>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <property name="horizontalSpacing">
                <number>3</number>
               </property>
               <property name="verticalSpacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="fieldLabel1">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Field</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="fieldComboBox1">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="fieldTypeLabel1">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel{ color: rgb(186, 189, 182); size: 8px}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="fieldFilterComboBox1">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="valueLabel1">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Value</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="valueLineEdit1">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="addFilterButton1">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border: none;
	background-color:  rgb(243, 243, 243);
}

QPushButton:hover{
	border: none;
	background-color:rgb(238, 238, 236);
}</string>
              </property>
              <property name="text">
               <string>Add Filter</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="checkStatus">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: white;
	background-color:white;
	border: 0px;
}</string>
            </property>
            <property name="text">
             <string>!</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QFormLayout" name="checkLayout">
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="1" column="1">
             <widget class="QComboBox" name="checkComboBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="currentText">
               <string/>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="checkLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Check</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="layerStatus2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: white;
	background-color:white;
	border: 0px;
}</string>
            </property>
            <property name="text">
             <string>!</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <layout class="QFormLayout" name="layerLayout2">
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="layerLabel2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Layer2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="layerComboBox2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="currentText">
               <string/>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="fieldFilterStatus2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: white;
	background-color:white;
	border: 0px;
}</string>
            </property>
            <property name="text">
             <string>!</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <layout class="QFormLayout" name="fieldFilterLayout2">
            <property name="verticalSpacing">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="fieldFilterLabel2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">color:white</string>
              </property>
              <property name="text">
               <string>Layer:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGroupBox" name="fieldFilterGroupBox2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="title">
               <string/>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="horizontalSpacing">
                <number>3</number>
               </property>
               <property name="verticalSpacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="fieldLabel2">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Field</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="fieldComboBox2">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="fieldTypeLabel2">
                 <property name="styleSheet">
                  <string notr="true">QLabel{ color: rgb(186, 189, 182); size: 8px}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="fieldFilterComboBox2">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="valueLabel2">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Value</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="valueLineEdit2">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QWidget:hover{
	border: none;
   border-bottom: 2px solid rgb(186, 189, 182); 
 }</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="addFilterButton2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border: none;
	background-color:  rgb(243, 243, 243);
}

QPushButton:hover{
	border: none;
	background-color:rgb(238, 238, 236);
}</string>
              </property>
              <property name="text">
               <string>Add Filter</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="createRuleButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border: none;
	background-color:  rgb(243, 243, 243);
}

QPushButton:hover{
	border: none;
	background-color:rgb(238, 238, 236);
}</string>
          </property>
          <property name="text">
           <string>Create rule</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultsLabel">
      <property name="text">
       <string>Results</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="resultList"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
